README.md

```diff
@@ -0,0 +1,30 @@
+# Installation
+This installation assumes you have a running instance of sensu-base and for ssl that you have your ssl-certs generated by the sensu-tool.
+```
+juju deploy sensu-client
+```
+The sensu-client requires 2 config options, password, which is the vhost password for the rabbitmq-server, exposed in juju status. The other one is rabbitmq, which is the privateip:port of the rabbitmq-instance. When using ssl, 2 extra config options must be set. Assuming you are in the sensu_ssl_tool directory:
```

Comment:
This is where cross-model relations are needed. Since setting up sensu per model is a huge waste of resources (running on google 1vCPU with 1.7 GB ram, redis had no cpu_load, rabbitmq 3% and sensu-base 3%), this is given through config options for now. But when cross-model relations become available, this would be rewritten (as mentioned in the remarks).